### PR TITLE
Dtspb 2808 reactivate dormant cases manual

### PR DESCRIPTION
**Pull request by itoulou**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2808

### Change description ###
manual event for case workers to reactivate dormant cases

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/controller/BusinessValidationController.java [hunk 1]
```diff
@@ -478,6 +478,14 @@ public ResponseEntity<CallbackResponse> defaultSolicitorNextStepsForPBANumbers(
             .transformCaseForSolicitorPBANumbers(callbackRequest, authToken));
     }
 
+    @PostMapping(path = "/reactivate-case", consumes = APPLICATION_JSON_VALUE, produces = {APPLICATION_JSON_VALUE})
```
Comment (nigeldunne) [inline src/main/java/uk/gov/hmcts/probate/controller/BusinessValidationController.java:515]: Add a unit test

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseState.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "BOCaseStoppedAwaitRedec", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "BORedecNotificationSent", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "BOSotGenerated", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "Dormant", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
```
Comment (nigeldunne) [inline ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseState.json:23]: Do we only want to add it for these two roles?

**Review by nigeldunne — CHANGES_REQUESTED** [review]

File: ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseState.json (hunk 2 above)
Comment (itoulou) [reply ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseState.json:23]: Possibly. Not sure this PR will be the one to deploy the new state though. I just needed it for this PR to work

File: src/main/java/uk/gov/hmcts/probate/controller/BusinessValidationController.java (hunk 1 above)
Comment (itoulou) [reply src/main/java/uk/gov/hmcts/probate/controller/BusinessValidationController.java:515]: test added

**Review by nigeldunne — APPROVED** [review]

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Pull request closed without merging**